src/coord/src/coord.rs

```diff
@@ -455,17 +493,38 @@ where
         // things the coordinator did without being related to the real dimension.
         let ts = (self.catalog.config().now)();
 
-        // We cannot depend on `self.catalog.config().now`'s value to increase
-        // (in addition to the normal considerations around clocks in computers,
-        // this feature enables us to drive the Coordinator's time when using a
-        // test harness). Instead, we must manually increment
-        // `last_open_local_ts` if `now` appears non-increasing.
-        self.last_open_local_ts = std::cmp::max(ts, self.last_open_local_ts + 1);
-
         // Opening a new timestamp means that there cannot be new writes at the
         // open timestamp.
         self.writes_at_open_ts = false;
```

Comment:
I think these toggles for the timestamping finite-state machine need different names because the notion of an "open timestamp" changes w/ this commit, i.e. sometimes there is no open timestamp, so it's unclear what a write at an open timestamp means, if the open timestamp value is essentially _null_. Maybe something more esoteric w/ some doc comments, like "writes_beyond_greatest_closed_ts".